.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@devamin I think that this configuration shouldn't be in the main branch, probably we have to exclude .vscode from the git ignore.